storm-core/src/clj/org/apache/storm/daemon/local_executor.clj

```diff
@@ -25,18 +25,3 @@
     (let [val (AddressedTuple. task tuple)]
```

Comment:
This entire file should really be removed and with-tracked-cluster should be updated to not use it.  It is obviously not needed because this function is never called, only over ridden.